Changes.md

```diff
@@ -5,6 +5,7 @@ Fixes
 -----
 
 - CyclesShader : Fixed broken presets menus.
+- OSLObject : Fixed `getattribute()` to support 64 bit integer data, such as an `instanceId` primitive variable loaded from USD.
```

Comment:
Should we be mentioning here that it is "supported" via truncation? This kind of makes it sound like you could read all 64 bits?

Reply:
Good point - I've added a note to that effect in the latest push.